tcat-app/inputparams.py

```diff
@@ -1,39 +1,57 @@
 # array structure [control_name , datatype , lookup_name , speaking_name , default_value , accepted_values]
 
 constellation_mission_params = {
-    ('mission_configuration', 'Mission Configuration'): [
-        ['select', 'text', 'propulsion_type', 'Propulsion type', 'electrical',
-         ['electrical', 'water', 'mono-propellant', 'solid', 'bi-propellant']],
+    ('scenario_metadata', 'Scenario Metadata'): [
```

Comment:
keep "mission configuration" (and the word "configuration" in all title, sorry Malo did not use the correct wording in the email)